src/python/pants/backend/graph_info/tasks/dependees.py

```diff
@@ -35,11 +35,11 @@ def __init__(self, *args, **kwargs):
   def console_output(self, _):
     address_mapper = self.context.address_mapper
     build_graph = self.context.build_graph
-
     dependees_by_target = defaultdict(set)
+    addresses = address_mapper.scan_addresses()
```

Comment:
So, actually... I think it should be possible to speed this up even further.

What you want to do is inject `::`, basically. And you can do that directly with: `graph.inject_specs_closure(DescendantAddresses('.'))`.